obsreport/obsreport_receiver.go

```diff
@@ -79,6 +79,14 @@ var (
 		receiverPrefix+RefusedMetricPointsKey,
 		"Number of metric points that could not be pushed into the pipeline.",
 		stats.UnitDimensionless)
+	mReceiverAcceptedLogRecords = stats.Int64(
+		receiverPrefix+AcceptedLogRecordsKey,
+		"Number of metric points successfully pushed into the pipeline.",
```

Comment:
cut & paste error: should be "log records", same below

Reply:
Done.